BackofficeBundle/Resources/views/BackOffice/Underscore/orchestraModalView._tpl.twig

```diff
@@ -1,15 +1,15 @@
 <div class="modal-dialog">
     <div class="modal-content">
-        <div class="modal-header">
+        <div class="modal-header clearfix">
             <button type="button" class="close" data-dismiss="modal">
                 <span aria-hidden="true">&times;</span>
                 <span class="sr-only">{{ ('open_orchestra_backoffice.modal.close') | trans() }}</span>
             </button>
+            <% if (typeof actionButtons != 'undefined') { %>
+            <%= actionButtons %>
+            <% } %>
```

Comment:
I'm not sure of what it looks in the browser: is the close button on the left side of the other buttons or on the right side ? The close button should be the last button on the right side.